sql/src/test/resources/calcite/expected/ingest/insertWithClusteredBy-logicalPlan.txt

```diff
@@ -1,4 +1,4 @@
-LogicalInsert(target=[druid.dst], partitionedBy=[{type=period, period=P1D, timeZone=UTC, origin=null}], clusteredBy=[2, `dim1`, CEIL(`m2`)])
+LogicalInsert(target=[druid.dst], partitionedBy=[FLOOR(`__time` TO DAY)], clusteredBy=[2, `dim1`, CEIL(`m2`)])
```

Comment:
Could we also a test with `PARTITIONED BY P1D` in the query so the plan would still contain the granularity?

Reply:
added